dts/msm8937-motorola-cedric.dts

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8952.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <294 0>;
+	qcom,board-id = <0x42 0x8500>;
```

Comment:
If I remember correctly cedric has more than one board-id it would be great to add them.